libs/api/domains/assets/src/lib/api-domains-assets.service.ts

```diff
@@ -36,127 +36,133 @@ export class AssetsXRoadService {
     auth: User,
     cursor?: string | null,
   ): Promise<any | null> {
-    const fasteignirResponse = await this.getRealEstatesWithAuth(
-      auth,
-    ).fasteignirGetFasteignir({
-      kennitala: auth.nationalId,
-      cursor: cursor,
-    })
+    try {
+      const fasteignirResponse = await this.getRealEstatesWithAuth(
+        auth,
+      ).fasteignirGetFasteignir({
+        kennitala: auth.nationalId,
+        cursor: cursor,
+      })
 
-    if (fasteignirResponse) {
-      return {
-        paging: fasteignirResponse.paging,
-        properties: fasteignirResponse.fasteignir?.map((item) => ({
-          propertyNumber: item.fasteignanumer,
-          defaultAddress: {
-            displayShort: item.sjalfgefidStadfang?.birtingStutt,
-            display: item.sjalfgefidStadfang?.birting,
-            propertyNumber: item.sjalfgefidStadfang?.landeignarnumer,
-            municipality: item.sjalfgefidStadfang?.sveitarfelagBirting,
-            postNumber: item.sjalfgefidStadfang?.postnumer,
-            locationNumber: item.sjalfgefidStadfang?.stadfanganumer,
-          },
-        })),
+      if (fasteignirResponse) {
+        return {
+          paging: fasteignirResponse.paging,
+          properties: fasteignirResponse.fasteignir?.map((item) => ({
+            propertyNumber: item.fasteignanumer,
+            defaultAddress: {
+              displayShort: item.sjalfgefidStadfang?.birtingStutt,
+              display: item.sjalfgefidStadfang?.birting,
+              propertyNumber: item.sjalfgefidStadfang?.landeignarnumer,
+              municipality: item.sjalfgefidStadfang?.sveitarfelagBirting,
+              postNumber: item.sjalfgefidStadfang?.postnumer,
+              locationNumber: item.sjalfgefidStadfang?.stadfanganumer,
+            },
+          })),
+        }
       }
+    } catch (e) {
```

Comment:
Is this catching 4xx error as well?

They might rather return `null` or `[]` since we're working inside a graphql context.